src/py/flwr/driver/driver.py

```diff
@@ -31,7 +31,13 @@
 class Driver:
     """`Driver` class provides an interface to the Driver API."""
 
-    def __init__(self) -> None:
+    def __init__(
+        self,
+        driver_service_address: str = DEFAULT_SERVER_ADDRESS_DRIVER,
+        certificates: Optional[bytes] = None,
```

Comment:
This should be documented in the docstring

Reply:
sure